### PR TITLE
Add support for globs

### PR DESCRIPTION
**Pull request by boneskull**
Hope you're interested in adding this.  Usually I'd ask first...but I didn't.

### CODE REVIEW

**Comment by boneskull** [comment]
My use case here is a recursive find/replace, but it needs to be portable

**Comment by sindresorhus** [comment]
Can you use https://github.com/sindresorhus/globby instead. It will handle the promise API, deduplication, and multiple patterns.

File: cli.js [hunk 1]
```diff
@@ -17,6 +17,7 @@ const cli = meow(`
 	Examples
 	  $ replace-in-files --string='horse' --regex='unicorn|rainbow' --replacement='🦄' foo.md
 	  $ replace-in-files --regex='v\\d+\\.\\d+\\.\\d+' --replacement=v$npm_package_version foo.css
+	  $ replace-in-files --string='blob' --replacement='blog' 'some/**/[gb]lob/*'
```
Comment (sindresorhus) [inline cli.js]: Would be good to show off that it supports multiple patterns and also negation.
```suggestion
	  $ replace-in-files --string='blob' --replacement='blog' 'some/**/[gb]lob/*' '!some/glob/foo'
```

**Comment by sindresorhus** [comment]
II think we also need to add a `--no-glob` flag for people that want to pass in pure paths which could potentially include something that is also a glob pattern.

**Comment by boneskull** [comment]
@sindresorhus OK, this is ready for review again.  I added `--no-glob` support and a test for this option.  Not sure what idiomatic Ava tests look like, but this is my best guess.

**Comment by sindresorhus** [comment]
Thanks ;)

**Pull request merged**